cypress/e2e/shared/loadDataSources.test.ts

```diff
@@ -152,32 +173,49 @@ describe.skip('Load Data Sources', () => {
     cy.getByTestID('lp-close--button').click()
 
     // navigate to data explorer to see data
+    // use new data explorer, since old data explorer is being deprecated
     cy.getByTestID('nav-item-data-explorer').click({force: true})
+
+    cy.getByTestID('script-query-builder-toggle').click()
+
     cy.getByTestID('timerange-dropdown').click()
-    cy.getByTestID('dropdown-item-customtimerange').click()
 
     // time range start
-    cy.getByTestID('timerange--input')
+    cy.get('.date-picker__input')
       .first()
       .clear()
       .type('2020-08-06 00:00:00.000')
 
     // time range stop
-    cy.getByTestID('timerange--input')
-      .last()
-      .clear()
-      .type('2020-08-08 00:00:00.000')
+    cy.get('.date-picker__input').last().clear().type('2020-08-08 00:00:00.000')
 
     cy.getByTestID('daterange--apply-btn').click()
 
-    // TODO replace this with proper health checks
     cy.wait(1000)
-    cy.get<string>('@defaultBucketListSelector').then(
-      (defaultBucketListSelector: string) => {
-        cy.getByTestID(defaultBucketListSelector).click()
-      }
-    )
+    cy.getByTestID('bucket-selector--dropdown-button').click()
+    cy.getByTestID('bucket-selector--dropdown--defbuck').click()
+
     // mymeasurement comes from fixtures/data.txt
-    cy.getByTestID('selector-list mymeasurement').should('exist')
+    cy.getByTestID('measurement-selector--dropdown-button').click()
+    cy.getByTestID('searchable-dropdown--item mymeasurement')
+      .contains('mymeasurement')
+      .click()
+  })
+})
+
+describe('Load Data Sources - New IOx', () => {
+  it('there is no old data explorer for new iox orgs', () => {
+    cy.skipOn(isTSMOrg)
+    cy.flush()
+    cy.signin()
+    cy.setFeatureFlags({
+      newDataExplorer: true,
+    })
+    cy.get('@org').then(({id}: Organization) =>
+      cy.fixture('routes').then(({orgs}) => {
+        cy.visit(`${orgs}/${id}/load-data/sources`)
+        cy.getByTestID('tree-nav')
```

Comment:
what is this testing exactly? Do we even need an IOx test for this particular test? The only thing different in load sources page for iox orgs is the number of client libraries, correct? If testing is necessary, maybe we can add a line that checks if the correct client library shows up?

Reply:
This was old cruft from another test - you're right that this just needs to be removed. Good catch!